packages/itwin/grouping-mapping-widget/src/widget/components/GroupPropertyAction.tsx

```diff
@@ -22,31 +22,54 @@ import {
 } from "@itwin/presentation-common";
 import { Presentation } from "@itwin/presentation-frontend";
 import { useActiveIModelConnection } from "@itwin/appui-react";
-import {
-  SvgChevronDown,
-  SvgChevronUp,
-  SvgRemove,
-} from "@itwin/itwinui-icons-react";
 import type { SelectOption } from "@itwin/itwinui-react";
 import {
   Alert,
+  Button,
   Fieldset,
   IconButton,
+  Label,
   LabeledInput,
   LabeledSelect,
-  Select,
+  Modal,
+  ModalButtonBar,
   Small,
+  Surface,
   Text,
 } from "@itwin/itwinui-react";
 import React, { useCallback, useEffect, useMemo, useState } from "react";
 import ActionPanel from "./ActionPanel";
 import useValidator, { NAME_REQUIREMENTS } from "../hooks/useValidator";
 import { handleError, WidgetHeader } from "./utils";
-import "./GroupPropertyAction.scss";
-import type { ECProperty, GroupProperty, GroupPropertyCreate} from "@itwin/insights-client";
-import { DataType, QuantityType } from "@itwin/insights-client";
 import { useMappingClient } from "./context/MappingClientContext";
 import { useGroupingMappingApiConfig } from "./context/GroupingApiConfigContext";
+import { HorizontalTile } from "./HorizontalTile";
+import { DataType, QuantityType } from "@itwin/insights-client";
+import type { ECProperty, GroupProperty, GroupPropertyCreate } from "@itwin/insights-client";
+import { SvgClose, SvgDragHandleVertical, SvgRemove, SvgSearch } from "@itwin/itwinui-icons-react";
```

Comment:
I get this error: `Module '"@itwin/itwinui-icons-react"' has no exported member 'SvgDragHandleVertical'.`

Reply:
Should be fixed.

Reply:
Should be fixed now really.